report_pre_thesis/src/sections/methodology.tex

```diff
@@ -206,4 +208,63 @@ \subsection{MOC}\label{sec:methodology_moc}
 \end{tabular*}
 \caption{Weighted Sum of Oxide Percentages. Elements marked with an asterisk (*) have been set to 50/50 as they are unspecified in \citet{cleggRecalibrationMarsScience2017}}
 \label{tab:weighted_sum_oxide}
-\end{table}
\ No newline at end of file
+\end{table}
+
+\subsection{Experiments}\label{sec:methodology_experiments}
+To evaluate the performance of each of the components in the pipeline, we focus our experiments on three main aspects:
+
+\begin{itemize}
+	\item \textbf{Outlier removal} to assess the impact of leaving outliers in the dataset or using a different outlier removal method.
+	\item \textbf{Hyperparameter tuning} to assess the impact of different hyperparameter configurations.
+	\item \textbf{Other models} to compare the performance of the PLS1-SM and ICA models to other models.
+\end{itemize}
+
+\noindent
+We recognize that these three aspects are commonly used to evaluate the performance and identify potential improvements of a machine learning pipeline.
```

Comment:
Commonly used to evaluate? Not really.

These are typically done during the explorative phase of a ML project, where you are trying to find the most problem model and dataset configuration. 
During such a stage, you'd evaluate many models on a validation set.
Few would likely pass your tests, at which point they would be considered for further optimization (e.g., extensive hyperparameter tuning, etc.).
Finally you'd get the end model's performance given some metric(s) - RMSE for us.

We need to be rather sharp when reasoning about our choices regarding experiments, given their importance to the project.

Reply:
Rephrased it, should be addressed now.